style.css

```diff
@@ -1,8 +1,144 @@
 body {
-  box-sizing: border-box;
   margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.nav__bar {
+  background: #64b5f6;
+  color: white;
+  overflow: auto;
+}
+
+.auth__name {
+  text-align: left;
+  margin-left: 2rem;
+}
+
+.links {
+  margin-right: 3rem;
+  text-align: right;
+  text-decoration: none;
+  color: white;
 
 }
-*, *:before, *:after {
-  box-sizing: inherit;
+
+.nav-right {
+  text-align: right;
+}
+
+.container {
+  /*grow up to 65rem*/
+  max-width: 70rem;
+  /*center if there is width left*/
+  margin: auto;
+}
+
+.card_left {
+  width: calc(100% / 2 - 1rem - 10rem);
```

Comment:
If you are using `card` as your grid item, then you'll want to use `margin` to make the space between different cards.

And all of the cards should be floated in the same direction instead of different directions.